src/NEO/Script.cpp

```diff
@@ -0,0 +1,26 @@
+// Copyright © 2017-2020 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+#include "Script.h"
+#include "OpCode.h"
+
+namespace TW::NEO {
+
+Data Script::CreateSignatureRedeemScript(Data publicKey) {
+  Data result;
+  result.push_back((byte)PUSHBYTES21);
+  result.insert(result.end(), publicKey.begin(), publicKey.end());
+  result.push_back((byte)CHECKSIG);
+  return result;
```

Comment:
Why only 2 space indent here?  In other place 4 is used.

Reply:
fixed